lib/active_data/model/validations.rb

```diff
@@ -18,6 +21,40 @@ def validate! context = nil
       def raise_validation_error
         raise ActiveData::ValidationError.new(self)
       end
+
+      module InstanceMethods
```

Comment:
Please check the comment above the line with `include InstanceMethods`. The method is overridden by `ActiveModel::Validations` if we put it directly to the module.